packages/next/src/build/webpack/loaders/next-edge-ssr-loader/render.ts

```diff
@@ -151,7 +152,10 @@ export function getRender({
 
   const handler = server.getRequestHandler()
 
-  return async function render(request: Request, event: NextFetchEvent) {
+  return async function render(
+    request: RequestWithFetchMetrics,
```

Comment:
We can use `WebNextRequest` from `packages/next/src/server/base-http/web.ts`, since it's for edge

Reply:
This would break typings all the way up. I'll open a separate PR to clean this up.